MAUI/DataForm/migration.md

```diff
@@ -0,0 +1,1085 @@
+
+<tr>
+<td>{{'[DataFormItems](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.DataFormGroupItem.html#Syncfusion_XForms_DataForm_DataFormGroupItem_DataFormItems)'| markdownify }}</td>
+<td>Items</td>
+<td>Gets or sets the value which used to set the items grouped under the group.</td>
+</tr>
+
+<tr>
+<td>{{'[IsExpanded](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.DataFormGroupItem.html#Syncfusion_XForms_DataForm_DataFormGroupItem_IsExpanded)'| markdownify }}</td>
+<td>IsExpanded</td>
+<td>Gets or sets a value indicating whether the group of editors is expanded or not.</td>
+</tr>
+
+<tr>
+<td>{{'[AllowExpandCollapse](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.DataFormGroupItem.html#Syncfusion_XForms_DataForm_DataFormGroupItem_AllowExpandCollapse)'| markdownify }}</td>
+<td>AllowExpandCollapse</td>
+<td>Gets or sets a value indicating whether users can collapse and expand the group of editors.</td>
+</tr>
+
+<tr>
+<td>Nil</td>
+<td>HeaderTextStyle</td>
+<td>Gets or sets the style of group header text, that used to customize the text color, font, font size, font family and font attributes.</td>
+</tr>
+
+<tr>
+<td>Nil</td>
+<td>HeaderBackground</td>
+<td>Gets or sets the background color for the header of the editor group.</td>
+</tr>
+
+<tr>
+<td>Nil</td>
+<td>ItemsPadding</td>
+<td>Gets or sets the distance between editors and the data form’s borders or group headers.</td>
+</tr>
+
+<tr>
+<td>{{'[GroupName](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.DataFormGroupItem.html#Syncfusion_XForms_DataForm_DataFormGroupItem_GroupName)'| markdownify }}</td>
+<td>Name</td>
+<td>Gets or sets the name of the editor group.</td>
+</tr>
+</table>
+
+### DataFormCheckBoxItem
+
+<table>
+<tr>
+<th>Xamarin SfDataForm</th>
+<th>.NET MAUI SfDataForm</th>
+<th>Description</th></tr>
+
+<tr>
+<td>{{'[CheckedColor](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.DataFormCheckBoxItem.html#Syncfusion_XForms_DataForm_DataFormCheckBoxItem_CheckedColor)'| markdownify }}</td>
+<td>Color</td>
+<td>Gets or sets the color of CheckBox editor.</td>
+</tr>
+</table>
+
+### DataFormSwitchItem
+
+<table>
+<tr>
+<th>Xamarin SfDataForm</th>
+<th>.NET MAUI SfDataForm</th>
+<th>Description</th></tr>
+
+<tr>
+<td>Nil</td>
+<td>OnColor</td>
+<td>Gets or sets the color of the switch when it is in the on state.</td>
+</tr>
+
+<tr>
+<td>Nil</td>
+<td>ThumbColor</td>
+<td>Gets or sets the thumb color of the switch editor.</td>
+</tr>
+</table>
+
+### DataFormTextEditorItem
+
+<table>
+<tr>
+<th>Xamarin SfDataForm</th>
+<th>.NET MAUI SfDataForm</th>
+<th>Description</th></tr>
+
+<tr>
+<td>{{'[KeyBoard](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.DataFormTextItemBase.html#Syncfusion_XForms_DataForm_DataFormTextItemBase_KeyBoard)'| markdownify }}</td>
+<td>Keyboard</td>
+<td>Gets or sets the type of keyboard used to display in the text, multiline and password editor.</td>
```

Comment:
<td>Gets or sets the type of keyboard used to display the text, multiline, and password editor.</td>